PowerEditor/installer/nativeLang/german.xml

```diff
@@ -873,7 +895,9 @@
 					<Item id="6120" name="Vertikal"/>
 					<Item id="6121" name="Beim Schließen des letzten Tabs beenden"/>
 
+                    <Item id="6131" name="Menü"/>
 					<Item id="6122" name="Menüleiste ausblenden (Umschalten mit Alt oder F10)"/>
+                    <Item id="6132" name="Rechte Tastenkürzel in der Menüleiste verbergen ＋ ▼ ✕ (Neustart von Notepad++ erforderlich)"/>
```

Comment:
"Tastenkürzel" does not fit to my understanding. They are either "Schaltflächen" or "Elemente"
**Proposal:** "Schaltflächen ＋ ▼ ✕ innerhalb der Menüleiste verbergen (Neustart von Notepad++ erforderlich)"

Reply:
Ok, good input. I've followed your suggestions.